app/dashboard/templates/bounty/fulfill.html

```diff
@@ -100,6 +100,13 @@ <h1 class="text-center title">{% trans "Submit Work" %}</h1>
     {% include 'shared/footer.html' %}
     {% include 'shared/messages.html' %}
   </body>
+
+  <!-- jQuery -->
```

Comment:
this is already loaded in footer_scripts.js

Reply:
1eb3739d21bc4c23760d2eae6fdcf96cc8d5bdab